src/user-event/paste.js

```diff
@@ -0,0 +1,71 @@
+import {getConfig as getDOMTestingLibraryConfig} from '../config'
+import {fireEvent, getActiveElement, calculateNewValue} from './utils'
+
+// this needs to be wrapped in the asyncWrapper for React's act and angular's change detection
+async function paste(...args) {
+  let result
+  await getDOMTestingLibraryConfig().asyncWrapper(async () => {
+    result = await pasteImpl(...args)
+  })
+  return result
+}
```

Comment:
I originally copied this from `type`, but I don't understand why `type` is no longer using `getConfig` and `asyncWrapper`. Should I keep this here, remove it, or use some higher level abstraction to make the function asynchronous?